src/ncurses_ui.cc

```diff
@@ -1106,4 +1149,15 @@ void NCursesUI::set_ui_options(const Options& options)
     }
 }
 
+int NCursesUI::get_zone_line (StringView zone_name)
```

Comment:
I dont like the use of `get_` here, should be `compute_` or just `zone_line`. Also, remove the space before `(`.